### DISCUSSION TITLE
Tree View Node

### DISCUSSION THREAD

**Discussion by wwderw** [Ethosa/nodesnim#42] [Ideas]
I was wondering if it would be possible for a treeview node?

I'm thinking something that could work for games or for traditional apps.

On the games end, could have it used for displaying inventory system and with regard for traditional apps, could be useful for SQLite programs.

Thoughts?

**Comment by Ethosa** [comment]
`TreeView` added in https://github.com/Ethosa/nodesnim/commit/4a9c8bfe462c0ed83fe1f71c4dfed4005cb5ae89 👀

**Comment by wwderw** [comment]
Cool.  Looking forward to trying that out.

Thanks!
